ecommerce/frontend/src/components/TrendingGrid.js

```diff
@@ -5,8 +5,8 @@ export const TrendingGrid = ({ trendingProducts }) => {
     return (
         <div className="gallery">
             {trendingProducts.map((product, index) => (
-                <figure className={'gallery__item gallery__item--' + index} key={product.productId}>
-                    <Link to={'/product/' + product.productId}>
+                <figure className={'gallery__item gallery__item--' + index} key={product.id}>
+                    <Link to={'/product/' + product.id}>
                         <img src={product.imageUrl} className="gallery__img" alt="" />
```

Comment:
shouldn't it be `product.image_url` with underscore? do i remember it incorrectly?